frontend/src/components/Pages/FrontPage/FrontPage.tsx

```diff
@@ -31,6 +32,7 @@ export function FrontPage() {
     return (
         <>
             <Header page={'root'} />
+            <SafeZoneBanner />
```

Comment:
Could it maybe be possible to set the banners to the header in the same way we do the other banners? Sorry that I haven't made it very obvious how to do that but there is an example on line 62 in ScheduleMissionDialog.tsx in the MissionOverview
![image](https://github.com/equinor/flotilla/assets/20266900/5fb484d2-62cf-4db0-ab90-982d5fffee1b)

Reply:
This alert is then displayed at the bottom of the Header.tsx file

![image](https://github.com/equinor/flotilla/assets/20266900/f8c3d03b-b57e-44bd-800e-4817ef9b5493)

Reply:
Yeah I can defiantly look into this, thanks!